### PR TITLE
added parse param decorator

### PR DESCRIPTION
**Pull request by ParallelTask**
Minor refactor

### CODE REVIEW

File: examples/parse_api/src/app/controllers/home.controller.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import {
+    ApiController,
+    Controller,
+    HttpGet,
+    SendsResponse,
```
Comment (TheWillG) [inline examples/parse_api/src/app/controllers/home.controller.ts]: Unused imports

**Comment by TheWillG** [comment]
Does this also work for query params (the example here shows path params)? If not, maybe we could think of a way to parse query params from web requests also.

File: examples/parse_api/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "dinoloop-http-verb-example",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "build": "npm run clean && tsc --p ./tsconfig.json",
+        "clean": "rimraf dist",
+        "lint": "tslint -p ./tsconfig.json -c ./tslint.json \"./src/**/*.ts*\"",
+        "serve": "node ./dist/examples/basic_start/src/app/app.js",
```
[the remark this answers is not in the document]
Comment (ParallelTask) [reply examples/parse_api/package.json]: Sure :)

File: examples/parse_api/src/app/controllers/home.controller.ts (hunk 1 above)
Comment (ParallelTask) [reply examples/parse_api/src/app/controllers/home.controller.ts]: Need to use smarter IDE

Comment (TheWillG) [reply examples/parse_api/src/app/controllers/home.controller.ts]: VS Code ESLint plugin should do the trick

File: src/modules/utility/dino.utility.ts [hunk 3]
```diff
@@ -20,7 +20,8 @@ import { ObjectUtility } from './object.utility';
 import { DataUtility } from './data.utility';
```
Comment (TheWillG) [inline src/modules/utility/dino.utility.ts:20]: L18 & L20 unused imports

**Review by TheWillG — APPROVED** [review]
Looks good overall. I think with these code changes we will be easily able to implement @ModelBinder and @QueryParamParse

**Pull request merged**